hack/util.sh

```diff
@@ -25,7 +25,7 @@ MIN_Go_VERSION=go1.16.0
 function util::install_tools() {
 	local package="$1"
 	local version="$2"
-
+  echo "go install ${package}@${version}"
```

Comment:
Maybe you can help to keep the space style the same. Seems some bad `tab`s here:)